LandSandBoat/server
# Fix Temenos crates disappearing soon after being spawned

**_I affirm:_**

## What does this pull request do?


This patch adds the MOBTYPE_BATTLEFIELD bit to poolid 7055, they were missed in 07b879387635884b9bc5efd3d02f1727a4526abf resulting in SetupBattlefieldMob() no longer being called on them.

Fixes #4175 

## Steps to test these changes


1. !pos 580.000 -2.375 104.000 37
2. !addkeyitem white_card
3. !addkeyitem cosmo_cleanse
4. Enter Temenos West, kill mobs until a crate spawns
5. Observe how it'll soon disappear without being opened (the first one takes a bit long)
6. Apply this patch
7. Observe how the crates no longer disappear without being opened
